src/crimson/os/seastore/seastore_types.h

```diff
@@ -1250,6 +1250,7 @@ struct extent_t {
 };
 
 using extent_version_t = uint32_t;
+constexpr extent_version_t EXTENT_VERSION_NULL = 0;
 
```

Comment:
Promote the increase-version method here. It's straitforward to understand how `extent_version_t` in a single place.